docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/vimeo-tracking/index.md

```diff
@@ -0,0 +1,186 @@
+---
```

Comment:
It really worries me that we have so much duplication. In any case, all comments above apply here as well. It would be great if you could make a single reusable page and import it into both of these pages. Reusable pages can have parameters. Feel free to slack me if you need help